src/java.desktop/share/classes/java/awt/FileDialog.java

```diff
@@ -140,15 +143,6 @@ public class FileDialog extends Dialog {
      @Serial
      private static final long serialVersionUID = 5035145889651310422L;
 
-
-    static {
-        /* ensure that the necessary native libraries are loaded */
-        Toolkit.loadLibraries();
```

Comment:
The same is here: I see no clear path to load the toolkit libraries from the peers, and peers do have native methods, including `initIDs`.

It looks as if peer implementations depend on the fact that the corresponding public API classes ensure the libraries are loaded.

Reply:
FileDialog extends Dialog, which loads libraries